projects/packages/jetpack-mu-wpcom/src/features/help-center/class-help-center.php

```diff
@@ -163,14 +163,14 @@ function () {
 			);
 		}
 
-		// `wp_set_script_translations` uses es_ES as the text domain, but we don't have translations for that.
-		// We want the locale to be 639 (i.e en, es, fr, etc).
-		add_filter( 'pre_determine_locale', array( $this, 'determine_iso_639_locale' ), 1 );
-
-		// phpcs:ignore Jetpack.Functions.I18n.TextDomainMismatch -- Jetpack is not responsible for the translate of this script.
-		wp_set_script_translations( 'help-center', 'default', plugin_dir_path( __FILE__ ) . 'languages' );
-
-		remove_filter( 'pre_determine_locale', array( $this, 'determine_iso_639_locale' ) );
+		// Load translations directly from widgets.wp.com.
+		wp_enqueue_script(
+			'help-center-translations',
```

Comment:
Does ordering matter?

With the old code, the `<script>` with the translations for the 'help-center' script would be output into the page first, then the `wp_add_inline_script( 'help-center', ..., 'before' )`, then the `<script>` for 'help-center' itself.

With this new code, I think there's no constraint on whether the 'help-center-translations' `<script>` comes before or after 'help-center'. If the ordering matters, adding 'help-center-translations' to the dependencies array of 'help-center' should be enough to make 'help-center-translations' come first.

Reply:
I tested and it does! Translations should be added first. I fixed that now.